.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: c
+sudo: required
+compiler: gcc
+dist: trusty
```

Comment:
This is the default now, any reason to pin it anyway?

Reply:
I'd like to avoid build errors triggered by changing defaults.